packages/vue-components/src/Navbar.vue

```diff
@@ -228,10 +249,14 @@ export default {
 
     // highlight current nav link
     this.highlightLink(window.location.href);
+
+    this.toggleLowerNavbar();
+    $(window).on('resize', this.toggleLowerNavbar);
   },
   beforeDestroy() {
     $('.dropdown', this.$el).off('click').offBlur();
     if (this.slots.collapse) $('[data-toggle="collapse"]', this.$el).off('click');
+    $(window).off();
```

Comment:
let's make it specific, so other listeners don't get removed

Reply:
Alright thanks. Updated :)